packages/@webex/plugin-meetings/src/reachability/util.ts

```diff
@@ -22,3 +22,25 @@ export function convertStunUrlToTurn(stunUrl: string, protocol: 'udp' | 'tcp') {
 
   return url.toString();
 }
+
+/**
+ * Converts a stun url to a turns url
+ *
+ * @param {string} stunUrl url of a stun server
+ * @returns {string} url of a turns server
+ */
+export function convertStunUrlToTurnTls(stunUrl: string) {
+  // stunUrl looks like this: "stun:external-media1.public.wjfkm-a-15.prod.infra.webex.com:443"
+  // and we need it to be like this: "turns:external-media1.public.wjfkm-a-15.prod.infra.webex.com:443?transport=tcp"
+  const url = new URL(stunUrl);
+
+  if (url.protocol !== 'stun:') {
+    throw new Error(`Not a STUN URL: ${stunUrl}`);
+  }
+
+  url.protocol = 'turns:';
+  url.port = '443'; // Make sure that the port is 443
```

Comment:
Just general comment, URL class is not changing port properly here because for some reason it's not working properly if protocol is not http/https.
Wondering if we should leave it anyway, just for informational purposes, or delete this line.

Reply:
ok, maybe remove it then so we don't have the confusion in unit tests. So looks like the only way to guarantee this to work would be to not use the URL class and just do string manipulations, probably not worth it as the server always sends us the port anyway

Reply:
Removed it for now